Resources/Prototypes/Entities/Objects/base_item.yml

```diff
@@ -36,3 +36,4 @@
     noRot: false
   - type: Pullable
   - type: ExplosionLaunched
+  - type: Forensics
```

Comment:
this should be dynamically added to the entity when interacted with, instead of being kept 100% of the time when it won't be used most of the time